Orange/widgets/visualize/owscatterplot.py

```diff
@@ -219,6 +224,11 @@ def reset_graph_data(self, *_):
     def set_data(self, data):
         self.information(1)
         self.__timer.stop()
+        for attr in data.domain:
+            if BasicStats(data[:, attr], attr).nans:
```

Comment:
I don't know about the efficiency of this, but the condition tests if the attribute contains _any_ nans. The [original issue](https://github.com/biolab/orange3/issues/1048) is about columns of _all_ nans.